Kinder/Registration.xaml.cs

```diff
@@ -25,6 +25,15 @@ public RegistrationWindow()
             InitializeComponent();
         }
 
+        public delegate void CheckDelegate(string Text);
+        
+        static void Checks (string text, CheckDelegate input, CheckDelegate field, CheckDelegate exists)
```

Comment:
Looks nice, but the fact is that we have a different number of validation checks in some "blocks", so what about other blocks, or its just a concept?

Reply:
Most blocks have 3 checks, but I want to check with the lecturer first if this is ok to count for the delegates requirement.